docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -84,6 +86,26 @@ gradlew eclipse
 . https://help.eclipse.org/2022-03/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm[Import the project to Eclipse]
 . Start creating your tests
 
+=== Visual Studio Code
+
+==== Prerequisites
+
+. Install https://code.visualstudio.com[Visual Studio Code]
+. Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studio[VIVIDUS Studio Extension]
+
+==== Open the project in Visual Studio Code
+
+. Open Visual Studio Code
+. Click `File` -> `Open...`
+. Open a folder with a VIVIDUS project
+. Wait until the message `Project with the name '...' is loaded` appears
+. Start creating your tests
+
+==== Execute created tests
+
+. Open https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette[Command Palette]
+. Type `VIVIDUS: Run Stories` in the search bar and select matched command
```

Comment:
and what's next?

Reply:
isn't it clear from the section title :) what would you like me to add here?

Reply:
what's happened next? how can users understand the tests run has started successfully?

Reply:
Do we really need this? the following are few actions from getting started guide:
* Install and configure Git - where is info on how to understand that the git was successfully installed?
* Build the project: - how the users understand that the build is successful?
* Execute the tests via CLI - how the users understand that tests has started successfully?
* ....